src/server/game/AuctionHouse/AuctionHouseMgr.h

```diff
@@ -76,11 +76,8 @@ struct AuctionEntry
     uint32 bidder;
     uint32 deposit;                                         //deposit can be calculated only when creating auction
     AuctionHouseEntry const* auctionHouseEntry;             // in AuctionHouse.dbc
-    uint32 factionTemplateId;
 
     // helpers
-    uint32 GetHouseId() const { return auctionHouseEntry->houseId; }
```

Comment:
The way I saw it, GetHouseId() existed before because it needed to actually lookup the value from the entry. Now it's actually already existing as exposed from the same level.

I changed all references to it. So, anyone now doing this knows to use houseId.

So, on this basis I'm disagreeing. But, I don't mind re-instating it. I don't think it makes a lot of difference.